exercises/J-array-get-set/exercises2.js

```diff
@@ -6,8 +6,10 @@
   - change the first value in the array to the number 1
 */
 
-let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+let numbers = [3, 2, 3]; // Don't change this array literal declaration
+// numbers.push(4);
+numbers[numbers.length] = 4;
```

Comment:
It's good that you are aware of 2 different ways of doing this. push(4) would also be perfectly correct.